IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MemberTypePost/DocPropertySet.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MemberTypePost" UniqueId="62bc6424-3d87-4aa3-a031-ae14e1e0bec8" ApplicableType="IfcMember/POST">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NominalHeight_1Fy4XYaJXDTBDDb9f06ofi" />
+		<DocProperty xsi:nil="true" href="ConicityRatio_1q2DJWtf58PvkMOQBrIqOF" />
+		<DocProperty xsi:nil="true" href="LoadBearingCapacity_2RBEe0qSKHuO00025QrEV" />
```

Comment:
This is described for floors, though not sure about it being on a product, would need description change. also this data is specific to the application direction and form, see comments on structural values below

Reply:
Yes! We have modify the description in other Pull Request. Should I modify def of bearing capacity here too ?